metaflow/plugins/kfp/tests/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM analytics-docker.artifactory.zgtools.net/artificial-intelligence/ai-platform/aip-py39-cpu:4.0.1380
 COPY . /home/zservice/metaflow
-RUN pip install --upgrade --no-deps --force-reinstall -e /home/zservice/metaflow
+RUN pip install --upgrade --force-reinstall -e /home/zservice/metaflow
```

Comment:
for my knowledge, why remove `--no-deps`?

Reply:
`--no-deps` prevents the installation of dependencies of `metaflow` like `click`, which Alex removed as manual dependencies.